src/Symfony/Component/HttpFoundation/Session/Storage/Proxy/SessionHandlerProxy.php

```diff
@@ -50,16 +50,16 @@ public function open($savePath, $sessionName)
     #[\ReturnTypeWillChange]
     public function close()
     {
-        return (bool) $this->handler->close();
+        return $this->handler->close();
```

Comment:
For this class, ref https://github.com/symfony/symfony/pull/41495, I don't understand why we need the casts.